pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -380,37 +398,29 @@ public enum PGProperty {
       "Specify 'options' connection initialization parameter."),
 
   /**
-   * Password to use when authenticating.
+   * Specifies the name of the file used to store passwords.
+   * Defaults to ~/.pgpass, or %APPDATA%\postgresql\pgpass.conf on Microsoft Windows.
+   * (No error is reported if this file does not exist.).
    */
-  PASSWORD(
-      "password",
+  PASSFILE(
+      "passfile",
       null,
-      "Password to use when authenticating.",
-      false),
+      "File used to store passwords (may be specified directly in the JDBC URL)"),
 
   /**
-   * Database name to connect to (may be specified directly in the JDBC URL).
+   * Password to use when authenticating.
    */
-  PG_DBNAME(
-      "PGDBNAME",
+  PASSWORD(
+      "password",
       null,
-      "Database name to connect to (may be specified directly in the JDBC URL)",
-      true),
-
-  /**
-   * Hostname of the PostgreSQL server (may be specified directly in the JDBC URL).
-   */
-  PG_HOST(
-      "PGHOST",
-      "localhost",
-      "Hostname of the PostgreSQL server (may be specified directly in the JDBC URL)",
+      "Password to use when authenticating.",
       false),
 
   /**
    * Port of the PostgreSQL server (may be specified directly in the JDBC URL).
    */
-  PG_PORT(
-      "PGPORT",
+  PORT(
+      "port",
```

Comment:
This looks like a change of public API to me.
Is it really needed?

Reply:
good catch. I wasn't particularly happy about changing the name of the property

Reply:
In case property names does not match there is need for conversion functions.
I had conversion functions in previous PR: https://github.com/MarekUniq/pgjdbc/blob/feature/urlparser-improve/pgjdbc/src/main/java/org/postgresql/util/PGPropertyUtil.java

and there is need to find all the places in the code where conversion functions should be called.

OR

maybe there can be both properties like this, then API remains backward compatible:
PG_PORT( "port" ...
PORT( "port" ...

Reply:
@davecramer I need your opinion here as well. Please read previous question.
There are 3 exceptional property names which does not match:
```
PG_DBNAME == "dbname"
PG_HOST == "host"
PG_PORT == "port"
```
I renamed them:
```
PG_DBNAME --> DBNAME 
PG_HOST --> HOST 
PG_PORT --> PORT 
```
I see your point. They are externally visible and it could be not backward compatible change in case anybody refers to them by consuming libraries.

**Option 1:**
In case we want to keep original names then we need to implement 2-way translation and find all the places it the code where translation takes place. This solution looks fragile and error prone to me.

**Option 2:**
Possibly we could keep existing property names but add 1-to-1 mappings as addition. Then API will be backward compatible. Optionally PG_ prefix names should become deprecated but does not have to.
```
PG_DBNAME == "dbname"
PG_HOST == "host"
PG_PORT == "port"
DBNAME == "dbname"
HOST == "host"
PORT == "port"
```

Reply:
My vote would be for option 2. Removing PG_HOST now would effect quite a few people.